projects/aca-shared/rules/src/app.rules.spec.ts

```diff
@@ -1003,6 +967,978 @@ describe('app.evaluators', () => {
       ).toBeFalse();
     });
   });
+
+  describe('isContentServiceEnabled', () => {
+    it('should call context.appConfig.get with correct parameters', () => {
+      const context: any = {
+        appConfig: {
+          get: jasmine.createSpy()
+        }
+      };
+
+      app.isContentServiceEnabled(context);
+      expect(context.appConfig.get).toHaveBeenCalledWith('plugins.contentService');
+    });
+
+    it('should return true if contentService plugin is enabled', () => {
+      expect(
+        app.isContentServiceEnabled({
+          appConfig: {
+            get: () => true
+          }
+        } as any)
+      ).toBeTrue();
+      expect(
+        app.isContentServiceEnabled({
+          appConfig: {
+            get: () => 'true'
+          }
+        } as any)
+      ).toBeTrue();
+    });
+
+    it('should return false if contentService plugin is disabled', () => {
+      expect(
+        app.isContentServiceEnabled({
+          appConfig: {
+            get: () => false
+          }
+        } as any)
+      ).toBeFalse();
+    });
+  });
+
+  describe('canCopyNode', () => {
+    const context = createTestContext();
```

Comment:
Can this line somehow be generalised in a beforeEach()/beforeAll() block, so that it's only mentioned once? The isContentServiceEnabled block is the only one where its not added, so maybe we can override the context there somehow instead

Reply:
Good point refactored the whole file to make each test case use context created in main beforeEach